compiler/shared/main/scala/mlscript/compiler/simpledef/Simpledef.scala

```diff
@@ -37,6 +43,20 @@ trait ProdObjImpl { self: ProdObj =>
   var objDestination: Set[ConsStrat] = Set()
 }
 
+class Context(
+  variables: Map[Var, ProdVar],
+  classes: Map[Var, ProdObj]) 
+{
```

Comment:
Trailing commas are better style:
```suggestion
  classes: Map[Var, ProdObj],
) {
```